examples/complete/lifecycle.us-east-2.tfvars

```diff
@@ -69,3 +69,5 @@ allowed_bucket_actions = [
   "s3:GetBucketLocation",
   "s3:AbortMultipartUpload"
 ]
+
+bucket_key_enabled = true
```

Comment:
Why is this specifically in the lifecycle rules fixture?

Reply:
Looks like you forgot to add `bucket_key_enabled` to the invocation of the root module in `examples/complete/main.tf`.

Reply:
Fixed.